pwa/static/configFiles/municipalities/hof-van-twente/hof-van-twente-accept.json

```diff
@@ -0,0 +1,15 @@
+{
+    "GATSBY_SHOW_THEME_SWITCHER": "false",
+    "GATSBY_API_BASE_URL": "https://hofvantwente.accept.openwoo.app/ap",
```

Comment:
You are missing the i in `api` it should be `https://hofvantwente.accept.openwoo.app/api`

Reply:
check!